pkg/api/types.go

```diff
@@ -2049,6 +2063,23 @@ type ServiceProxyOptions struct {
 	Path string
 }
 
+// OwnerReference contains enough information to let you identify an owning
+// object. Currently, an owning object must be in the same namespace, so there
+// is no namespace field.
+type OwnerReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
```

Comment:
I am mostly questioning forcing a version to be specified (by combining the group and version) when what I am mostly likely to want to specify is just a group (and let the garbage collector watch the server's preferred version or pick a version containing that kind)